repository/repository-api/src/main/java/org/eclipse/vorto/repository/api/attachment/AttachResult.java

```diff
@@ -0,0 +1,69 @@
+package org.eclipse.vorto.repository.api.attachment;
+
+import org.eclipse.vorto.repository.api.ModelId;
+
+public class AttachResult {
+	private boolean success;
```

Comment:
AttachmentResult contains fields that are already in the Attachment object. I would suggest to have a reference to attachment objects.